src/dotnet/commands/dotnet-restore/Program.cs

```diff
@@ -194,9 +194,11 @@ private static string GenerateProjectJsonContents(IEnumerable<string> frameworks
             foreach (var framework in frameworks)
             {
                 var importsStatement = "\"imports\": [ \"dnxcore50\", \"portable-net452+win81\" ]";
-                
                 sb.AppendLine($"        \"{framework}\": {{ {importsStatement} }}");
             }
+            sb.AppendLine("    },");
+            sb.AppendLine("    \"runtimes\": { ");
+            sb.AppendLine($"        \"{DefaultRid}\": {{}}");
```

Comment:
/cc @piotrpMSFT @brthor Seemed like the easiest way to replicate the intended behavior since NuGet has removed `--runtime` (for now at least)